runtime/browser/android/xwalk_content.cc

```diff
@@ -467,6 +468,22 @@ jboolean XWalkContent::SetManifest(JNIEnv* env,
     // No need to display launch screen, load the url directly.
     Java_XWalkContent_onGetUrlFromManifest(env, obj, url_buffer.obj());
   }
+  std::string view_background_color;
+  ManifestGetString(manifest,
+                    keys::kXWalkViewBackgroundColor,
+                    keys::kViewBackgroundColor,
+                    &view_background_color);
+
+  if (view_background_color.empty())
+    return true;
+  unsigned int view_background_color_int = 0;
+  if (!base::HexStringToUInt(view_background_color.substr(1),
```

Comment:
In the "xwalk_launch_screen", it use color with prefix "#". We do not want to re-invent a wheel.